client/dashboard/profile-wizard/steps/business-details.js

```diff
@@ -36,6 +37,16 @@ class BusinessDetails extends Component {
 		const { addNotice, goToNextStep, isError, updateProfileItems } = this.props;
 		const { other_platform, product_count, selling_venues } = this.state;
 
+		const alreadySelling = [ 'other', 'brick-mortar-other', 'brick-mortar' ].includes(
```

Comment:
Could we check if `selling_venues` is just blank or equal to `no`?  Probably set this to `no` as I found a bug here that needs fixing - https://github.com/woocommerce/woocommerce-admin/issues/2533

Reply:
Haha yeah that's better. Fixed in https://github.com/woocommerce/woocommerce-admin/pull/2531/commits/957e23e92a6131ff847b2e2c800be7a1b7880ebd, thanks!